setup/res/index.php.txt

```diff
@@ -64,8 +64,58 @@ $classLoader = implode(DIRECTORY_SEPARATOR, [$civiCorePath, 'CRM', 'Core', 'Clas
 require_once $classLoader;
 CRM_Core_ClassLoader::singleton()->register();
 
+function standaloneErrorHandler(
+    int $errno,
+    string $errstr,
+    ?string $errfile,
+    ?int $errline,
+    ?array $errcontext) {
+  static $handlingError = FALSE;
+  if ($handlingError) {
+    throw new \RuntimeException("Died: error was thrown during error handling");
+  }
+  $config = CRM_Core_Config::singleton();
+  if (!$config->debug) {
+    // For these errors to show, we must be debugging.
+    return;
+  }
+
+  $handlingError = TRUE;
+  $trace = '';
+  if ($config->backtrace) {
+    // Backtrace is configured for errors.
+    $trace = [];
+    foreach (array_slice(debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS), 1) as $item) {
+      $_ = '';
+      if (!empty($item['function'])) {
+        if (!empty($item['class']) && !empty($item['type'])) {
+          $_ = htmlspecialchars("$item[class]$item[type]$item[function]() ");
+        }
+        else {
+          $_ = htmlspecialchars("$item[function]() ");
+        }
+      }
+      $_ .= "<code>" . htmlspecialchars($item['file']) . '</code> line ' . $item['line'];
+      $trace[] = $_;
+    }
+    $trace = '<pre class=backtrace>' . implode("\n", $trace) . '</pre>';
+  }
+
+  if (!isset(Civi::$statics[__FUNCTION__])) {
+    Civi::$statics[__FUNCTION__] = [];
+  }
+  Civi::$statics[__FUNCTION__][] = '<li style="white-space:pre-wrap">'
```

Comment:
`Civi::$statics` here doesn't seem to be used as a cache? I know there's been a trend towards using longer variable names but I don't think that was the intent here (grin).

Reply:
Oh nevermind. I see.